flake.nix

```diff
@@ -2,13 +2,12 @@
   description = "Chainweb";
 
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs?rev=4d2b37a84fad1091b9de401eb450aae66f1a741e";
     hackage = {
       url = "github:input-output-hk/hackage.nix";
       flake = false;
     };
-    haskellNix = {
-      url = "github:input-output-hk/haskell.nix";
+    hs-nix-infra = {
+      url = "github:kadena-io/hs-nix-infra/enis/metadata-experiments";
```

Comment:
We intend to follow the `main` branch at this repo, yes?

Reply:
@jwiegley, that's correct, I'm just experimenting a few approaches for now.